arcade/infection/infection_amusement_park/map.xml

```diff
@@ -1,131 +1,132 @@
-    </kit>
-</kits>
-<spawns>
-    <spawn team="human-team" kit="default-human-kit" yaw="180" safe="true" filter="before-30s" region="humans-spawn-point"/>
-    <spawn team="zombie-team" kit="infected-kit-start" yaw="180" filter="before-30s" region="infected-spawn-point"/>
-    <spawn team="human-team" kit="zombie-switch-kit" yaw="180" safe="true" filter="after-30s" region="humans-spawn-point"/>
-    <spawn team="zombie-team" kit="infected-kit" yaw="180" filter="after-30s" region="infected-spawn-point"/>
-    <default region="obs-spawn-point"/>
-</spawns>
-<!-- TODO: migrate fake destroyables to structure or fill actions -->
-<destroyables name="cage" required="false" show="false" owner="human-team">
-    <destroyable materials="stained glass" region="infected-spawn" modes="start-game"/>
-</destroyables>
-<modes show-before="25s">
-    <mode id="start-game" after="25s" material="air" name="`3${zombie-team-name} are being released"/>
-</modes>
-<filters>
-    <not id="before-30s">
-        <time id="after-30s">31s</time>
-    </not>
-    <material id="only-door">wooden door</material>
-</filters>
-<regions>
-    <cylinder id="obs-spawn-point" base="50,28,-120" height="0" radius="2"/>
-    <point id="infected-spawn-point">23.5,5,35.5</point>
-    <cylinder id="humans-spawn-point" base="25.5,9,25.5" height="0" radius="2"/>
-    <cuboid id="infected-spawn" min="21,4,32" max="27,9,38"/>
-    <union id="door-region">
-        <block>48,10,-119</block>
-        <block>48,11,-119</block>
-        <block>47,10,-132</block>
-        <block>47,11,-132</block>
-        <block>51,9,-136</block>
-        <block>51,10,-136</block>
-        <block>45,11,-127</block>
-        <block>45,12,-127</block>
-        <block>26,6,-87</block>
-        <block>26,7,-87</block>
-        <block>27,3,-86</block>
-        <block>27,4,-86</block>
-    </union>
-    <apply region="door-region" use="deny(only-door)"/>
-</regions>
-<damage>
-    <deny>
-        <attacker>
-            <filter id="before-30s"/>
-        </attacker>
-    </deny>
-</damage>
-<disabledamage>
-    <damage>drowning</damage>
-</disabledamage>
-<kill-rewards>
-    <kill-reward filter="only-humans">
-        <kit>
-            <effect duration="5" amplifier="2">regeneration</effect>
-        </kit>
-    </kill-reward>
-</kill-rewards>
-<itemremove>
-    <item>leather helmet</item>
-    <item>leather chestplate</item>
-    <item>leather leggings</item>
-    <item>leather boots</item>
-    <item>iron sword</item>
-    <item>bow</item>
-    <item>arrow</item>
-</itemremove>
-</map>
+<map proto="1.5.0">
+<name>Infection: Amusement Park</name>
+<variant id="halloween" world="halloween">Halloween Edition</variant>
+<version>1.1.4</version>
+<created>2022-04-12</created><!--created date derived from commit 0a44b860-->
+<include id="infection"/>
+<if variant="default">
+    <created>2022-04-01</created>
+</if>
+<if variant="halloween">
+    <created>2022-10-03</created>
+</if>
```

Comment:
uhh... i think your script needs a bit of tweaking, this map already has creation date(s)

Reply:
Yea, same with Hide n Seek. Probably anything with the created in an <if> got included?